core/vm/evm.go

```diff
@@ -145,10 +145,13 @@ type EVM struct {
 	// abort is used to abort the EVM calling operations
 	// NOTE: must be set atomically
 	abort int32
-	// callGasTemp holds the gas available for the current call. This is needed because the
+	// CallGasTemp holds the gas available for the current call. This is needed because the
 	// available gas is calculated in gasCall* according to the 63/64 rule and later
 	// applied in opCall*.
-	callGasTemp uint64
+	CallGasTemp uint64
+	// callErrorTemp holds any errors caused during the execution of system opcodes (0xf0)
+	// NOTE: it's being used only for tracers
+	CallErrorTemp error
```

Comment:
We have checked for any EIPs that explicitly set that VM errors are not allowed in tracers as they might leak out any information, and we found nothing. Are there any reasons for not returning the errors in the first place?